docs/UserGuide.md

```diff
@@ -29,7 +29,7 @@ It is a convenient platform for you to keep track of your lessons and assignment
 
    Some example commands you can try:
 
-   * **`list`** : Lists all assignments.
+   * **`list 2`** : Lists all assignments and lessons.
```

Comment:
User command should be just "list" instead of "list 2" to fit the explanation fo the command. Possible typo?

Reply:
Changed!

Reply:
Looks good!